### DISCUSSION TITLE
Transfer Google Photos Favorites and Descriptions to Immich

### DISCUSSION THREAD

**Discussion by jjspierx** [immich-app/immich#5520] [General]
Does anyone know of a way to transfer descriptions and favorites from Google Photos images to Immich?  Using Google Takeout, I have all of the JSON sidecars which contain the descriptions and if a photo has been favorited.  Where does Immich store this information for photos?  Is it stored as meta data in EXIF of the photos, or stored in the DB?  Could I write a script that takes the information from the JSON sidecar, and inserts this into Immich so my descriptions and favorited photos show up the same in Immich?

**Comment by bo0tzz** [comment]
Immich stores descriptions in the database, you could update it with a call to the [`updateAsset`](https://immich.app/docs/api/update-asset) endpoint.

@simulot maybe this would be a nice addition to immich-go?

**Reply by simulot** [reply]
Indeed! 
The idea is to give the description and favorite during the import.
For those having already imported their collection, immich-go could update existing assets.

I'm wondering why it's not possible to give the description, and latitude / longitude during the initial uploadFile call

**Reply by simulot** [reply]
https://github.com/simulot/immich-go/discussions/93

**Comment by jjspierx** [comment]
I had not heard of immich-go until just now.  Really cool looking tool!
